tests/Unit/DataStructures/Test_VectorImplHelper.hpp

```diff
@@ -68,50 +63,45 @@ constexpr inline void apply_tuple_combinations(
 }
 //@}
 
-// @{
+namespace detail {
+/// \cond HIDDEN_SYMBOLS
```

Comment:
nothing in a detail namespace shows up in documentation, so no need for `cond`. More of a general note for you :)